bin/alpen-cli/src/seed.rs

```diff
@@ -122,6 +124,7 @@ impl Seed {
     }
 }
 
+#[expect(missing_debug_implementations)]
```

Comment:
It should be fine if this is `Debug`, since it's encrypted. But also fine to leave it out as an extra layer of security (albeit by obscurity).